erppeek.py

```diff
@@ -16,16 +16,18 @@
 import sys
 import time
 import traceback
-try:                    # Python 3
+
+PY2 = (sys.version_info[0] == 2)
```

Comment:
We could use the `six` library instead of `sys`:

```python
from six import PY2
```

It detects which version of python is running.